scripts/e2e.ts

```diff
@@ -115,8 +124,10 @@ class E2ETestRunner extends events.EventEmitter {
       // Set feature flags
       process.env.RD_ENV_EXTENSIONS = '1';
 
+      // Start the renderer process and wait for it to complete the build.
+      await this.startRendererProcess();
```

Comment:
It sounds like this is no longer "starting the renderer process", and instead it's just "build the renderer completely".  It would be clearer to rename the function, I think.